src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -196,6 +197,22 @@ public String toSQL() {
 	class RowMap extends HashMap<String, Object> {
 		private final HashMap<String, Object> data;
 
+		@Override
+		public Set<String> keySet() {
```

Comment:
Does this even need to be a `LinkedHashSet`? Why not just `ArrayList` or even dumb `String[]`?